tutorials/getting_started/getting_started.md

```diff
@@ -0,0 +1,183 @@
+```
+
+Here `accuracy` measures how often our model predicted the "correct" part of speech tag as most probable,
+while `accuracy3` measures how often the correct tag was one of the _three_ most probable.
+(`loss` measures [cross entropy](https://en.wikipedia.org/wiki/Cross_entropy)
+ and is the objective being used to train the model. You want to make sure
+ it's mostly decreasing during training.)
+
+After 20 epochs we see we see
+
+```
+2017-08-15 13:22:01,591 - INFO - allennlp.training.trainer - Epoch 20/20
+accuracy: 0.85, loss: 0.58, accuracy3: 0.92 ||: 100%|##########| 477/477 [00:25<00:00, 19.75it/s]
+accuracy: 0.71, loss: 1.38, accuracy3: 0.83 ||: 100%|##########| 50/50 [00:01<00:00, 47.64it/s]
+2017-08-15 13:22:28,214 - INFO - allennlp.training.trainer - Training accuracy : 0.849197    Validation accuracy : 0.714408
+2017-08-15 13:22:28,214 - INFO - allennlp.training.trainer - Training loss : 0.577224    Validation loss : 1.378879
+2017-08-15 13:22:28,214 - INFO - allennlp.training.trainer - Training accuracy3 : 0.924702    Validation accuracy3 : 0.831475
+2017-08-15 13:22:28,222 - INFO - allennlp.training.trainer - Best validation performance so far. Copying weights to /tmp/tutorials/getting_started/best.th'.
+```
+
+This means that 71% of the time our model predicted the correct tag on the validation dataset,
+and 83% of the time the correct tag was in the model's "top 3".
+(Not ground-breaking performance, but this is a pretty simple model, and
+if you look at the data there's a lot of different tags!)
+
+Now that the model is trained, there should be a bunch of files in the serialization directory. The `vocabulary` directory
+contains the model's vocabularies, each of which is a (distinct) encoding of strings as integers.
+In our case, we'll have one for `tokens` (i.e. words) and another for `tags`. The various
+`training_state_epoch_XX.th` files contain the state of the trainer after each epoch (`.th` is the suffix for serialized torch tensors),
+so that you could resume training where you left off, if you wanted to.
+Similarly, the `model_state_epoch_XX.th` files contain the model weights after each epoch.
+Finally `best.th` contains the *best* weights (that is, those from the epoch with the smallest `loss`).
+
+### Evaluating a Model
+
+Once you've trained a model, you likely want to evaluate it on another dataset.
+In [tutorials/getting_started/data/cp.test](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/data/cp.test)
+we have the contents of Brown Corpus `cp01` (the first file in the "Romance" section).  As our configuration file specifies where
+the model was serialized to, it also tells us enough to evaluate the model:
+
+```
+allennlp/run evaluate --config_file tutorials/getting_started/simple_tagger.json --evaluation_data_file tutorials/getting_started/data/cp.test
+```
+
+After again spitting out a bunch of parameter values, it will evaluate the trained model on the evaluation data file:
```

Comment:
Might make sense to disable parameter and initialization logging in the `evaluate` command.  I did this in `common/testing/test_case.py`, if you want to disable the most verbose logging statements with a copy/paste.